package.json

```diff
@@ -1,58 +1,77 @@
 {
-    "name": "mag-ah-frontend",
-    "version": "0.1.0",
-    "private": true,
-    "dependencies": {
-        "node-sass": "^4.9.3",
-        "react": "^16.5.2",
-        "react-dom": "^16.5.2",
-        "react-router-dom": "^4.3.1",
-        "react-redux": "^5.0.7",
-        "redux": "^4.0.0",
-        "redux-thunk": "^2.3.0",
-        "react-scripts": "2.0.3"
-    },
-    "scripts": {
-        "start": "react-scripts start",
-        "build": "react-scripts build",
-        "test": "react-scripts test",
-        "eject": "react-scripts eject",
-        "lint": "eslint . --ext .js,.jsx",
-        "lintfix": "eslint . --fix"
-    },
-    "eslintConfig": {
-        "extends": "react-app"
-    },
-    "browserslist": [
-        ">0.2%",
-        "not dead",
-        "not ie <= 11",
-        "not op_mini all"
-    ],
-    "devDependencies": {
-        "eslint-config-airbnb": "^17.1.0",
-        "eslint-config-prettier": "^3.1.0",
-        "eslint-config-prettier-standard": "^2.0.0",
-        "eslint-config-standard": "^12.0.0",
-        "eslint-plugin-import": "^2.14.0",
-        "eslint-plugin-jsx-a11y": "^6.1.1",
-        "eslint-plugin-node": "^7.0.1",
-        "eslint-plugin-prettier": "^3.0.0",
-        "eslint-plugin-promise": "^4.0.1",
-        "eslint-plugin-react": "^7.11.1",
-        "eslint-plugin-standard": "^4.0.0",
-        "husky": "^1.1.0",
-        "pre-commit": "^1.2.2",
-        "prettier": "^1.14.3",
-        "prettier-eslint": "^8.8.2",
-        "lint-staged": "^7.3.0"
-    },
-    "pre-commit": [
-        "lint"
-    ],
-    "lint-staged": {
-        "*.js": [
-            "prettier --write"
-        ]
-    }
+  "name": "mag-ah-frontend",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "axios": "^0.18.0",
+    "bootstrap": "^4.1.3",
+    "expect": "^23.6.0",
+    "fetch-mock": "^7.0.7",
+    "jquery": "^3.3.1",
```

Comment:
Could we remove jquery from dependencies?

Reply:
@pngg bootsrap requires jquery as a dependency. We need it.